.github/workflows/ci.yml

```
@@ -7,10 +7,9 @@ on:
     branches: ["master"]
 
 jobs:
-  tox:
+  test:
```

Hmmh this was perhaps not exactly fortunate, as the `id` seems to be defined in repository branch protection settings as required checks… in that case feel free to revert this back:

```suggestion
  tox:
```